### PR TITLE
Fix CheckBox on Mac

### PR DESCRIPTION
**Pull request by vladest**
https://highfidelity.manuscript.com/f/cases/11009/Mac-Dynamic-checkbox-when-importing-entities-with-asset-browser-misaligned-or-missing

Test plan:
- Open Asset Browser on Mac
- Check if all checkboxes aligned

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7702/app_12553_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7702/HighFidelity-Beta-PR12553-12a70d57c2903ccdd37fbfc7ccc7733b977a232d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7702/HighFidelity-Beta-PR12553-12a70d57c2903ccdd37fbfc7ccc7733b977a232d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7703/app_12553_PR.apk

File: interface/resources/qml/controls-uit/CheckBox.qml [hunk 1]
```diff
@@ -101,11 +102,11 @@ Original.CheckBox {
...
         label: Label {
-            text: control.text
-            color: control.color
+            text: checkBox.text
+            color: checkBox.color
             x: 2
-            wrapMode: checkBox.wrap ? Text.Wrap : Text.NoWrap
-            elide: checkBox.wrap ? Text.ElideNone : Text.ElideRight
+            //wrapMode: checkBox.wrap ? Text.Wrap : Text.NoWrap
+            //elide: checkBox.wrap ? Text.ElideNone : Text.ElideRight
```
Comment (sethalves) [inline interface/resources/qml/controls-uit/CheckBox.qml]: is there a reason to leave these commented out, here?

Comment (ystelmahovskaya) [reply interface/resources/qml/controls-uit/CheckBox.qml]: its not final version yet

Comment (vladest) [reply interface/resources/qml/controls-uit/CheckBox.qml]: sorry, answered from wife's Mac book. 
its not for merge yet, but at least I able to reproduce it now

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7703/HighFidelity-Beta-PR12553-1d43ca48bd77241d222fbce38eb6a62cf578f946.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7703/HighFidelity-Beta-PR12553-1d43ca48bd77241d222fbce38eb6a62cf578f946.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7704/app_12553_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7704/HighFidelity-Beta-PR12553-9f151566bf30e58c51e064ac0320395c0df899b4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7704/HighFidelity-Beta-PR12553-9f151566bf30e58c51e064ac0320395c0df899b4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7705/app_12553_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7705/HighFidelity-Beta-PR12553-c2c7962f66d8d6f94304d86158463343ce6946af.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7705/HighFidelity-Beta-PR12553-c2c7962f66d8d6f94304d86158463343ce6946af.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7706/app_12553_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7706/HighFidelity-Beta-PR12553-43e7f282ec562cfb0e4f9ba2fd0c22c2978954fe.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7706/HighFidelity-Beta-PR12553-43e7f282ec562cfb0e4f9ba2fd0c22c2978954fe.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Checkbox for dynamic is now lined up and looked as expected.

**Pull request merged**